perch-agent/src/agent/action_store.py

```
@@ -0,0 +1,145 @@
+    async def start(self) -> None:
+        """Start the background TTL eviction loop. Idempotent."""
+        if self._gc_task is None or self._gc_task.done():
+            self._gc_task = asyncio.create_task(self._gc_loop())
+
+    async def stop(self) -> None:
+        if self._gc_task is not None:
+            self._gc_task.cancel()
+            try:
+                await self._gc_task
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(asyncio.CancelledError, Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(asyncio.CancelledError, Exception): ...`

(SIM105)